docs-additonal/development-setup.md

```diff
@@ -0,0 +1,59 @@
+# Development Setup for Cobbler-Web
+
+## Basic setup
+
+This will give you a setup of both Repositories of the main Git branches.
+
+1. Clone Cobbler: `git clone git@github.com:cobbler/cobbler.git`
+2. Clone Cobbler Web: `git clone git@github.com:cobbler/cobbler-web.git`
+3. Go into the Cobbler directory and execute run the following steps:
+   - Build Docker Image: `docker build -f docker/testing/testing.dockerfile -t cobbler-testing .`
+   - Run built image: `docker run -d --name cobbler-testing -v $PWD:/code cobbler-testing`
+   - Inspect the container and gets its IP: `docker inspect cobbler-testing | grep "IPAddress"`
+4. Go into the Cobbler-Web directory and follow the following steps:
+   - Adjust the file `projects/cobbler-frontend/proxy.conf.json` and replace in the line with `target` the string
+     `localhost` with the just filtered IP-Address.
+   - Build the TS-XMLRPC API via: `npm run build typescript-xmlrpc`
+   - Build the Cobbler-API project via: `npm run build cobbler-api`
+   - Serve the Cobbler frontend via: `npm run start cobbler-frontend`
+5. Go to <http://localhost:4200> to see the frontend.
```

Comment:
```suggestion
This will give you a setup of both repositories of the main Git branches.

1. Clone Cobbler: `git clone git@github.com:cobbler/cobbler.git`
2. Clone Cobbler Web: `git clone git@github.com:cobbler/cobbler-web.git`
3. Go into the `cobbler` directory and execute the following steps:
   - Build the Docker image: `docker build -f docker/testing/testing.dockerfile -t cobbler-testing .`
   - Run the built image: `docker run -d --name cobbler-testing -v $PWD:/code cobbler-testing`
   - Inspect the container and its IP address: `docker inspect cobbler-testing | grep "IPAddress"`
4. Go into the `cobbler-web` directory and follow these steps:
   - Adjust the file `projects/cobbler-frontend/proxy.conf.json` and in the line with `target` replace the string
     `localhost` with the just filtered IP address.
   - Build the TS-XMLRPC API via: `npm run build typescript-xmlrpc`
   - Build the Cobbler-API project via: `npm run build cobbler-api`
   - Serve the Cobbler frontend via: `npm run start cobbler-frontend`
5. Go to <http://localhost:4200> to see the frontend.
```